resources/elasticsearch/ds_cellar_mapping.json

```diff
@@ -264,6 +264,12 @@
             "ignore_above": 256
           }
         }
+      },
+      "eu_cellar_core": {
```

Comment:
unfortunately, we need these fields here (only for the ds_eu_cellar dataset).
The current solution can be hardcoded, but we need to move slowly towards DSL models for ElasticSearch.